models/DataObject/ClassDefinition/Data/ManyToManyRelation.php

```diff
@@ -699,6 +699,38 @@ public function getFromWebserviceImport($value, $relatedObject = null, $params =
         }
     }
 
+    /**
+     * TODO: move validation to checkValidity & throw exception in Pimcore 7
+     * @param DataObject\Concrete|DataObject\Localizedfield|DataObject\Objectbrick\Data\AbstractData|\Pimcore\Model\DataObject\Fieldcollection\Data\AbstractData $object
+     * @param array $params
+     */
+    public function save($object, $params = [])
+    {
+        if (!DataObject\AbstractObject::isDirtyDetectionDisabled() && $object instanceof DataObject\DirtyIndicatorInterface) {
```

Comment:
As this code is 100% identical with the one in `ManyToManyObjectRelation` they should share it. 
What about a trait `Pimcore\Model\DataObject\ClassDefinition\Data\Relations\ManyToManyRelationTrait` that has this `save()` method?

Reply:
done!